HISTORY.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 0.2.3 (2020-05-19)
+
+- Refactor removal of top-level directory after cookiecutting to avoid collisions. (See [#15](https://github.com/zillow/battenberg/pull/15))
+
 ## 0.x.x (YYYY-MM-DD)
```

Comment:
Mind just merging these two releases, I think I fixed another issue previously but didn't bother to release it may as well release it now :)

Reply:
no worries! ill move it in.